git/homework.py

```diff
@@ -78,7 +83,15 @@ def multiple_ints_with_conversion(first_value: Any, second_value: Any) -> int:
             print("Not valid input data")
         >>> "Not valid input data"
     """
-    pass
+    if type(second_value) == string:
+        if not second_value.isdigit():
+            raise ValueError
+
+    if type(first_value) == string:
```

Comment:
you can use only one condition for two variables

Reply:
Like this?

if type(second_value) and type(first_value) == string:
  raise ValueError